### PR TITLE
Build navigation bar page and modal page

### PR DESCRIPTION
**Pull request by ishitakeshawani**
Preview link : https://deploy-preview-6--felizui.netlify.app/

Screenshot (102)

### CODE REVIEW

**Review by ranitasahaa1230 — APPROVED** [review]
Looks good to me

File: JS/modal.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+var modal = document.getElementById("myModal");
+
+var btn = document.getElementById("myBtn");
+
+var span = document.getElementsByClassName("close-btn")[0];
+
```
Comment (anupkgurung) [inline JS/modal.js]: this might give an undefined if the class you are looking for might not available.

**Review by anupkgurung — APPROVED** [review]
Extra spaces can be removed. rest look good !!

File: JS/modal.js (hunk 1 above)
Comment (ishitakeshawani) [reply JS/modal.js]: yes

**Pull request merged**